src/features/rewards/disabledBox/index.tsx

```diff
@@ -0,0 +1,36 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this file,
+* You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+import * as React from 'react'
+import {
+  StyledWrapper,
+  StyledCard,
+  StyledTitle,
+  StyledText
+} from './style'
+import { getLocale } from '../../../helpers'
+
+export default class DisabledBox extends React.PureComponent<{}, {}> {
+  render () {
+    return (
+      <StyledWrapper>
+        <StyledCard>
+          <StyledTitle>
+            {getLocale('whyBraveRewards')}
+          </StyledTitle>
+          <StyledText padding={true}>
```

Comment:
why is text padding from title different?

Reply:
The desired effect is now done with CSS, no props